### PR TITLE
Add additional label support for Post Type / Taxonomy register functions

### LINKED ISSUE #3269: Add additional label support for Post Type / Taxonomy register functions
Post Type:

- [x] feature_image ("Featured Image")
- [x] set_featured_image ("Set featured image")
- [x] remove_featured_image ("Remove featured image")
- [x] use_featured_image ("Use as featured image")
- [x] archives ("Post Archives")
- [x] insert_into_item ("Insert into post" / "Insert into page")
- [x] uploaded_to_this_item ("Uploaded to this post" / "Uploaded to this page")
- [x] filter_items_list ("Filter posts lists" / "Filter pages list")
- [x] items_list_navigation ("Posts list navigation" / "Pages list navigation")
- [x] items_list ("Posts list" / "Pages list")
- [x] Confirm other labels are covered from https://developer.wordpress.org/reference/functions/get_post_type_labels/

Taxonomy:

- [x] no_terms ("No tags" / "No categories")
- [x] items_list_navigation ("Tags list navigation" / "Categories list navigation")
- [x] items_list ("Tags list" / "Categories list")
- [x] Confirm other labels are covered from https://developer.wordpress.org/reference/functions/get_taxonomy_labels/

https://make.wordpress.org/core/2015/12/11/additional-labels-for-custom-post-types-and-custom-taxonomies/

### PR DESCRIPTION
**Pull request by pcfreak30**
Fixes #3269

### CODE REVIEW

File: classes/PodsInit.php [hunk 1]
```diff
@@ -849,6 +849,16 @@ public static function object_label_fix ( $args, $type = 'post_type' ) {
             $labels[ 'not_found_in_trash' ] = pods_var_raw( 'not_found_in_trash', $labels, sprintf( __( 'No %s Found in Trash', 'pods' ), $label ), null, true );
             $labels[ 'parent' ] = pods_var_raw( 'parent', $labels, sprintf( __( 'Parent %s', 'pods' ), $singular_label ), null, true );
             $labels[ 'parent_item_colon' ] = pods_var_raw( 'parent_item_colon', $labels, sprintf( __( 'Parent %s:', 'pods' ), $singular_label ), null, true );
+            $labels[ 'feature_image' ] = pods_var_raw( 'add_new', $labels, __( 'Featured Image', 'pods' ), null, true );
```
Comment (sc0ttkclark) [inline classes/PodsInit.php]: 'add_new' should be using the correct label key here.

File: classes/PodsInit.php [hunk 2]
```diff
@@ -849,6 +849,16 @@ public static function object_label_fix ( $args, $type = 'post_type' ) {
...
+            $labels[ 'set_featured_image' ] = pods_var_raw( 'add_new', $labels, __( 'Set featured image', 'pods' ), null, true );
```
Comment (sc0ttkclark) [inline classes/PodsInit.php]: 'add_new' should be using the correct label key here.

File: classes/PodsInit.php [hunk 3]
```diff
@@ -849,6 +849,16 @@ public static function object_label_fix ( $args, $type = 'post_type' ) {
...
+            $labels[ 'remove_featured_image' ] = pods_var_raw( 'add_new', $labels, __( 'Remove featured image', 'pods' ), null, true );
```
Comment (sc0ttkclark) [inline classes/PodsInit.php]: 'add_new' should be using the correct label key here.

File: classes/PodsInit.php [hunk 4]
```diff
@@ -849,6 +849,16 @@ public static function object_label_fix ( $args, $type = 'post_type' ) {
...
+            $labels[ 'use_featured_image' ] = pods_var_raw( 'add_new', $labels, __( 'Use as featured image', 'pods' ), null, true );
```
Comment (sc0ttkclark) [inline classes/PodsInit.php]: 'add_new' should be using the correct label key here.

File: classes/PodsInit.php [hunk 5]
```diff
@@ -849,6 +849,16 @@ public static function object_label_fix ( $args, $type = 'post_type' ) {
...
+            $labels[ 'archives' ] = pods_var_raw( 'archives', $labels, sprintf( __( '%s Archives', 'pods' ), $singular_label ), null, true );
+            $labels[ 'insert_into_item' ] = pods_var_raw( 'insert_into_item', $labels, sprintf( __( 'Insert into %s', 'pods' ), $singular_label ), null, true );
+            $labels[ 'uploaded_to_this_item' ] = pods_var_raw( 'uploaded_to_this_item', $labels, sprintf( __( 'Uploaded to this %s', 'pods' ), $singular_label ), null, true );
+            $labels[ 'filter_items_list' ] = pods_var_raw( 'filter_items_list', $labels, sprintf( __( 'Filter %s lists', 'pods' ), $singular_label ), null, true );
+            $labels[ 'items_list_navigation' ] = pods_var_raw( 'items_list_navigation', $labels, sprintf( __( 'Posts %s navigation', 'pods' ), $singular_label ), null, true );
+            $labels[ 'items_list' ] = pods_var_raw( 'items_list_navigation', $labels, sprintf( __( 'Posts %s', 'pods' ), $singular_label ), null, true );
```
Comment (sc0ttkclark) [inline classes/PodsInit.php]: 'items_list_navigation' should be using the correct label key here.

File: classes/PodsInit.php [hunk 6]
```diff
@@ -864,6 +874,9 @@ public static function object_label_fix ( $args, $type = 'post_type' ) {
             $labels[ 'separate_items_with_commas' ] = pods_var_raw( 'separate_items_with_commas', $labels, sprintf( __( 'Separate %s with commas', 'pods' ), $label ), null, true );
             $labels[ 'add_or_remove_items' ] = pods_var_raw( 'add_or_remove_items', $labels, sprintf( __( 'Add or remove %s', 'pods' ), $label ), null, true );
             $labels[ 'choose_from_most_used' ] = pods_var_raw( 'choose_from_most_used', $labels, sprintf( __( 'Choose from the most used %s', 'pods' ), $label ), null, true );
+            $labels[ 'no_terms' ] = pods_var_raw( 'no_terms', $labels, sprintf( __( 'No %s', 'pods' ), $singular_label ), null, true );
+            $labels[ 'items_list_navigation' ] = pods_var_raw( 'items_list_navigation', $labels, sprintf( __( 'Posts %s navigation', 'pods' ), $singular_label ), null, true );
+            $labels[ 'items_list' ] = pods_var_raw( 'items_list_navigation', $labels, sprintf( __( 'Posts %s', 'pods' ), $singular_label ), null, true );
```
Comment (sc0ttkclark) [inline classes/PodsInit.php]: 'items_list_navigation' should be using the correct label key here.

File: classes/PodsInit.php [hunk 7]
```diff
@@ -849,6 +849,16 @@ public static function object_label_fix ( $args, $type = 'post_type' ) {
...
+            $labels[ 'remove_featured_image' ] = pods_var_raw( 'add_new', $labels, __( 'Remove featured image', 'pods' ), null, true );
+            $labels[ 'use_featured_image' ] = pods_var_raw( 'add_new', $labels, __( 'Use as featured image', 'pods' ), null, true );
+            $labels[ 'archives' ] = pods_var_raw( 'archives', $labels, sprintf( __( '%s Archives', 'pods' ), $singular_label ), null, true );
+            $labels[ 'insert_into_item' ] = pods_var_raw( 'insert_into_item', $labels, sprintf( __( 'Insert into %s', 'pods' ), $singular_label ), null, true );
+            $labels[ 'uploaded_to_this_item' ] = pods_var_raw( 'uploaded_to_this_item', $labels, sprintf( __( 'Uploaded to this %s', 'pods' ), $singular_label ), null, true );
+            $labels[ 'filter_items_list' ] = pods_var_raw( 'filter_items_list', $labels, sprintf( __( 'Filter %s lists', 'pods' ), $singular_label ), null, true );
+            $labels[ 'items_list_navigation' ] = pods_var_raw( 'items_list_navigation', $labels, sprintf( __( 'Posts %s navigation', 'pods' ), $singular_label ), null, true );
```
Comment (sc0ttkclark) [inline classes/PodsInit.php]: Needs correct "%s list navigation" label default

File: classes/PodsInit.php (hunk 5 above)
Comment (sc0ttkclark) [inline classes/PodsInit.php]: Needs correct "%s list" label default

File: classes/PodsInit.php [hunk 8]
```diff
@@ -864,6 +874,9 @@ public static function object_label_fix ( $args, $type = 'post_type' ) {
             $labels[ 'separate_items_with_commas' ] = pods_var_raw( 'separate_items_with_commas', $labels, sprintf( __( 'Separate %s with commas', 'pods' ), $label ), null, true );
             $labels[ 'add_or_remove_items' ] = pods_var_raw( 'add_or_remove_items', $labels, sprintf( __( 'Add or remove %s', 'pods' ), $label ), null, true );
             $labels[ 'choose_from_most_used' ] = pods_var_raw( 'choose_from_most_used', $labels, sprintf( __( 'Choose from the most used %s', 'pods' ), $label ), null, true );
+            $labels[ 'no_terms' ] = pods_var_raw( 'no_terms', $labels, sprintf( __( 'No %s', 'pods' ), $singular_label ), null, true );
```
Comment (sc0ttkclark) [inline classes/PodsInit.php]: Needs to use `$label`

File: classes/PodsInit.php [hunk 9]
```diff
@@ -864,6 +874,9 @@ public static function object_label_fix ( $args, $type = 'post_type' ) {
...
+            $labels[ 'items_list_navigation' ] = pods_var_raw( 'items_list_navigation', $labels, sprintf( __( 'Posts %s navigation', 'pods' ), $singular_label ), null, true );
```
Comment (sc0ttkclark) [inline classes/PodsInit.php]: Needs correct "%s list navigation" label default

File: classes/PodsInit.php [hunk 10]
```diff
@@ -849,6 +849,16 @@ public static function object_label_fix ( $args, $type = 'post_type' ) {
...
+            $labels[ 'feature_image' ] = pods_var_raw( 'add_new', $labels, __( 'Featured Image', 'pods' ), null, true );
+            $labels[ 'set_featured_image' ] = pods_var_raw( 'add_new', $labels, __( 'Set featured image', 'pods' ), null, true );
+            $labels[ 'remove_featured_image' ] = pods_var_raw( 'add_new', $labels, __( 'Remove featured image', 'pods' ), null, true );
+            $labels[ 'use_featured_image' ] = pods_var_raw( 'add_new', $labels, __( 'Use as featured image', 'pods' ), null, true );
+            $labels[ 'archives' ] = pods_var_raw( 'archives', $labels, sprintf( __( '%s Archives', 'pods' ), $singular_label ), null, true );
+            $labels[ 'insert_into_item' ] = pods_var_raw( 'insert_into_item', $labels, sprintf( __( 'Insert into %s', 'pods' ), $singular_label ), null, true );
+            $labels[ 'uploaded_to_this_item' ] = pods_var_raw( 'uploaded_to_this_item', $labels, sprintf( __( 'Uploaded to this %s', 'pods' ), $singular_label ), null, true );
+            $labels[ 'filter_items_list' ] = pods_var_raw( 'filter_items_list', $labels, sprintf( __( 'Filter %s lists', 'pods' ), $singular_label ), null, true );
```
Comment (sc0ttkclark) [inline classes/PodsInit.php]: Needs to use `$label` instead of `$singular_label`

File: classes/PodsInit.php (hunk 7 above)
Comment (sc0ttkclark) [inline classes/PodsInit.php]: Needs to use `$label` instead of `$singular_label`

File: classes/PodsInit.php (hunk 5 above)
Comment (sc0ttkclark) [inline classes/PodsInit.php]: Needs to use `$label` instead of `$singular_label`

File: classes/PodsInit.php (hunk 9 above)
Comment (sc0ttkclark) [inline classes/PodsInit.php]: Needs to use `$label` instead of `$singular_label`

File: classes/PodsInit.php (hunk 6 above)
Comment (sc0ttkclark) [inline classes/PodsInit.php]: Needs to use `$label` instead of `$singular_label`

File: ui/admin/setup-edit.php [hunk 11]
```diff
@@ -378,7 +378,46 @@
...
+    <div class="pods-field-option">
+        <?php echo PodsForm::label( 'label_remove_featured_image', __( 'Remove featured image', 'pods' ), __( 'help', 'pods' ) ); ?>
+        <?php echo PodsForm::field( 'label_remove_featured_image', pods_var_raw( 'label_remove_featured_image', $pod ), 'text' ); ?>
+    </div>
+    <div class="pods-field-option">
+        <?php echo PodsForm::label( 'label_use_featured_image', __( 'Use as featured image', 'pods' ), __( 'help', 'pods' ) ); ?>
+        <?php echo PodsForm::field( 'label_use_featured_image', pods_var_raw( 'label_use_featured_image', $pod ), 'text' ); ?>
+    </div>
+    <div class="pods-field-option">
+        <?php echo PodsForm::label( 'label_archives', __( 'Post Archives', 'pods' ), __( 'help', 'pods' ) ); ?>
```
Comment (sc0ttkclark) [inline ui/admin/setup-edit.php]: Needs to use singular item placeholder in label

File: ui/admin/setup-edit.php [hunk 12]
```diff
@@ -820,7 +863,7 @@
                     <div class="submitbox" id="submitpost">
                         <div id="major-publishing-actions">
                             <div id="delete-action">
-                                <a href="<?php echo esc_url( pods_query_arg( array( 'action' . $obj->num => 'delete', '_wpnonce' => wp_create_nonce( 'pods-ui-action-delete' ) ) ) ); ?>" class="submitdelete deletion pods-confirm" data-confirm="<?php _e( 'Are you sure you want to delete this Pod? All fields and data will be removed.', 'pods' ); ?>"> <?php _e( 'Delete', 'pods' ); ?> Pod </a>
+                                <a href="<?php echo esc_url( pods_query_arg( array( 'action' . $obj->num => 'delete' ) ) ); ?>" class="submitdelete deletion pods-confirm" data-confirm="<?php _e( 'Are you sure you want to delete this Pod? All fields and data will be removed.', 'pods' ); ?>"> Delete Pod </a>
```
Comment (sc0ttkclark) [inline ui/admin/setup-edit.php]: What caused this change? Can you revert this line?

**Comment by pcfreak30** [comment]
Commit bug fixed. Was a huge copy/paste fail with CTRL + D in phpstorm.

File: ui/admin/setup-edit.php (hunk 12 above)
Comment (pcfreak30) [inline ui/admin/setup-edit.php]: This is based off the 2.x branch. I did not changed that line.

Comment (sc0ttkclark) [inline ui/admin/setup-edit.php]: This is a PR against 2.x, it says your code changed what is in 2.x currently, may have just been some weird issue but let's get it back to the previous line.

Comment (pcfreak30) [inline ui/admin/setup-edit.php]: This has been fixed. I accidentally copied from a older version.

File: classes/PodsInit.php [hunk 13]
```diff
@@ -849,6 +849,16 @@ public static function object_label_fix ( $args, $type = 'post_type' ) {
...
+            $labels[ 'use_featured_image' ] = pods_var_raw( 'use_featured_image', $labels, __( 'Use as featured image', 'pods' ), null, true );
+            $labels[ 'archives' ] = pods_var_raw( 'archives', $labels, sprintf( __( '%s Archives', 'pods' ), $singular_label ), null, true );
+            $labels[ 'insert_into_item' ] = pods_var_raw( 'insert_into_item', $labels, sprintf( __( 'Insert into %s', 'pods' ), $singular_label ), null, true );
+            $labels[ 'uploaded_to_this_item' ] = pods_var_raw( 'uploaded_to_this_item', $labels, sprintf( __( 'Uploaded to this %s', 'pods' ), $singular_label ), null, true );
+            $labels[ 'filter_items_list' ] = pods_var_raw( 'filter_items_list', $labels, sprintf( __( 'Filter %s lists', 'pods' ), $label ), null, true );
+            $labels[ 'items_list_navigation' ] = pods_var_raw( 'items_list_navigation', $labels, sprintf( __( '%s navigation', 'pods' ), $label ), null, true );
+            $labels[ 'items_list' ] = pods_var_raw( 'items_list_navigation', $labels, sprintf( __( '%s list', 'pods' ), $label ), null, true );
```
Comment (sc0ttkclark) [inline classes/PodsInit.php]: Needs to reference `items_list`

**Comment by sc0ttkclark** [comment]
Can merge into Pods 2.6 as soon as the two remaining issues are resolved.

**Pull request merged**